.github/workflows/ci.yml

```
@@ -0,0 +1,41 @@
+name: Continuous Integration
+
+on: ['push', 'pull_request']
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: true
+      matrix:
+        elixir: [1.9.x, 1.8.x, 1.7.x]
+        otp: [22.3, 21.2, 19.x]
```

```suggestion
        otp: [22.3, 21.2, 19.2]
```
After check the log of the build, I think the reason of the fail is the OTP v19.x in the `otp` matrix. In the README of [actions/setup-elixir](https://github.com/actions/setup-elixir) there is a note that seems relevant for this case:

> Note The OTP release version specification is relatively complex. For best results, we recommend specifying exact OTP and Elixir versions. However, values like 22.x are also accepted, and we attempt to resolve them according to semantic versioning rules.

So let's try with the version 19.2, although this one doesn't seem available on Ubuntu Packages repository. I introduce the test for Erlang 19 it is the newer version supported on Debian 9 (Stretch), and I was interested in this testing personally although I use a newer version of Debian and have Erlang installed by another way, not the Debian Package.

In the case the build will not run after this change, it will probably be necessary to remove this test due it would be because there is not Erlang 19 on Ubuntu Packages repository.

---

@owenvoke, I am going to perform the suggestion to check if it works, and if not works, begin to think about an alternative or if it is necessary to remove this version from the tests.

---

I did originally try with v17.3 (which is one of the versions used in Travis), but that failed with an incompatible warning. 🤷

---

I am going to review it deeply. I will notice you any new about it.